PR: Add blog post recapping and featuring Quansight Spyder webcast (OSD Ep. 1)

CAM-Gerlach opened the pull request:
A fairly short and sweet post that features the video from the recent webcast. Hopefully the embed works; we should definitely verify at the very least when we merge to develop. Should be posted as soon as it is checked and approved, since we also want to get the 3.3.1 post out as soon as the conda packages hit the defaults channel in a few days time.

Thanks!

ccordoba12 commented on content/blog/webcast-quansight-ep1/contents.lr:
```diff
@@ -0,0 +1,30 @@
+title: Spyder featured on Episode 1 of Open Source Directions web show
+---
+pub_date: 2018-08-12 00:00
```
Note: Don't forget to update date when finally published.

CAM-Gerlach replied:
Hopefully that will be soon :)

ccordoba12 commented on content/blog/webcast-quansight-ep1/contents.lr:
```diff
@@ -0,0 +1,30 @@
+[Quansight](https://www.quansight.com/), the company recently founded by NumPy, SciPy and Anaconda creator Travis Oliphant to help connect companies with open source communities built around data science and machine learning, just released Episode 1 of its live webcast series, and it was all about Spyder!
+Spyder maintainer [Carlos Córdoba](https://github.com/ccordoba12), recently hired by Quansight and funded part-time to work on Spyder development [as we announced a few weeks ago](https://www.spyder-ide.org/blog/spyder-status-2018-present/), was the featured guest on the show.
+
+Carlos first shared his perspective some of the key moments in Spyder's nearly 10-year development history, from its original creation by Pierre Raybaut and his initial involvement in the project to its [more recent challenges and successes](https://www.spyder-ide.org/blog/spyder-status-2018-past/).
```
... his perspective *about* some ...

Seems to be missing here.

jitseniesen replied:
The second "his" seems to refer to Pierre. However, I can't find an easy to reformulate and it is pretty clear in context that it in fact refers to Carlos, so perhaps it be left as is

CAM-Gerlach replied:
Yeah, that's a typo; thanks--too late at night (err, morning) to be typing lol. Changed to "on".

CAM-Gerlach replied:
I changed "his" to "Carlos'". Its less clean, yes, but more clear and it doesn't read *that* bad I don't think.

CAM-Gerlach commented on content/blog/webcast-quansight-ep1/contents.lr:
```diff
@@ -0,0 +1,30 @@
+
+<iframe width="640" height="360" src="https://www.youtube-nocookie.com/embed/vtLdH4VbACA" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+We'll have a new post on the release of Spyder 3.3.1 (and 3.3.0) in a few days time, plus posts on our new docs, Spyder 4 beta 1, and our full roadmap all in the next few weeks, so there's plenty going on you won't want to miss.
```
Also, if this post gets delayed enough that the release post has to go out first, then I'll need to update this line.

CAM-Gerlach replied:
Also, note to self: Change one "post" to "article" to avoid word repetition.

ccordoba12 replied:
I think we can have this ready today.

CAM-Gerlach replied:
:)

ccordoba12 commented on content/blog/webcast-quansight-ep1/contents.lr:
```diff
@@ -0,0 +1,30 @@
+
+Carlos first shared his perspective some of the key moments in Spyder's nearly 10-year development history, from its original creation by Pierre Raybaut and his initial involvement in the project to its [more recent challenges and successes](https://www.spyder-ide.org/blog/spyder-status-2018-past/).
+He also demonstrated basic usage of Spyder, as well as some of its standout features, in a live on-screen demo.
+Carlos  then went on to outline the current roadmap for Spyder 4 in the near future, as well as detail the key new features planned and under development for the upcoming major version.
```
* Carlos  then ...

    Please remove the additional blank between those two words.

* Is this part of video ok? Because I was thinking to record myself talking about the main features of our Roadmap. What do you think @CAM-Gerlach?

ccordoba12 replied:
And then upload that part of the video to Youtube too.

jitseniesen replied:
For me, "the current roadmap" is largely the same as "the key new features", and I'm not sure everybody realizes that "the upcoming major version" is the same as "Spyder 4", so I'd write this as: "Carlos then went on to outline the current roadmap for Spyder 4 in the near future, explaining (or: detailing) the key new features planned and under development."

CAM-Gerlach replied:
> Please remove the additional blank between those two words.

Yup, fixed.

>  I was thinking to record myself talking about the main features of our Roadmap. What do you think @CAM-Gerlach?

I think that's a fantastic idea! What's in there is basically them awkwardly reading off the brochure we did so it would be way better, both in terms of being more infomative and more compelling, to have you actually explaining the key new features and the motivation behind them. Your mic setup was pretty good for the purpose (and audio is much more important than video in this context); video wise the background was fine but if you can get way more light on your face, that would be great since you were almost a silhouette.

CAM-Gerlach replied:
@jitseniesen I ended up just doing a full re-writing of the sentence to 

> Carlos then went on to outline the current roadmap for Spyder 4 in the near future, and explained some of the key new features planned for it.

since what I'd previously written was sufficiently bad to require even more major surgery than suggested, lol.

ccordoba12 replied:
Ok, I'll record myself later today and upload the video. I'll post the link when it's ready.

jitseniesen commented on content/blog/webcast-quansight-ep1/contents.lr:
```diff
@@ -0,0 +1,30 @@
+Finally, he took the time to answer a variety of Spyder-related questions asked live by viewers, ranging from specific current and planned Spyder features to suggestions on keeping a good work-life balance.
+While technical difficulties (since identified and resolved) interrupted some of his commentary, particularly in the roadmap section, a full account of the latter will be published right here in the coming weeks once Spyder 4 beta 1 is released.
+
+If you missed the webcast, Quansight recorded it and uploaded it to their [new Youtube channel](https://www.youtube.com/channel/UChdlbCpQ_Wep04V9o0sGLWQ), so you can watch it on-demand right here if you're curious about any of the above.
```
I'd say "right there", not "right here"

CAM-Gerlach replied:
It's ``right here`` because the video is in fact included right in the post a few lines below, embedded in an ``iframe``, rather than just linked to (where ``there`` might indeed be more appropriate).

CAM-Gerlach commented:
@ccordoba12 Addressed all the feedback received. If/when this gets merged to ``develop``, we shouldn't forget to check the video renders in and looks decent before deploying to ``production``.

ccordoba12 commented:
Unfortunately, I don't have time to do the Roadmap video today either. Since I think we shouldn't let to pass much time to promote this, we should merge and release this post.

@CAM-Gerlach, what do you think?

ccordoba12 commented:
> We could either have it as a separate post

I think this is better because this is a simple and straightforward post. Final word?

CAM-Gerlach commented:
@ccordoba12 Agreed.

ccordoba12 commented:
Thanks! Merging then.

ccordoba12 commented:
This looks good, so I'll merge it with production.
